package.json

```diff
@@ -3,10 +3,10 @@
   "description": "media atom editor",
   "scripts": {
     "build-autotrack": "autotrack -o public/video-ui/build/autotrack.js -p cleanUrlTracker,eventTracker,impressionTracker,maxScrollTracker,mediaQueryTracker,outboundFormTracker,outboundLinkTracker,pageVisibilityTracker,socialWidgetTracker,urlChangeTracker",
-    "build": "webpack -p ./public/video-ui/src/app.js ./public/video-ui/build/app.js --config ./build_config/webpack.prod.conf.js",
-    "build-dev": "webpack ./public/video-ui/src/app.js ./public/video-ui/build/app.js --config ./build_config/webpack.dev.conf.js --watch",
+    "build": "webpack -p ./public/video-ui/src/app.js -o ./public/video-ui/build/app.js --config ./build_config/webpack.prod.conf.js",
+    "build-dev": "webpack -o ./public/video-ui/build/app.js --config ./build_config/webpack.dev.conf.js --watch",
     "client-dev": "node ./webpack-dev-server.js",
```

Comment:
Adding `-o` specifies that the second arg is the output, which looks like it wasn't the case in webpack v1. If you omit the `-o` it would just compile the build directory as well, meaning you're "building" the build again.